src/main/kotlin/com/perflog/domain/review/service/ReviewService.kt

```diff
@@ -53,7 +58,12 @@ class ReviewService(
     }
 
     fun getReviewsByPerfume(perfumeId: Long): List<ReviewDto.Response> {
-        return reviewRepository.findByPerfumeId(perfumeId).map { review ->
+        val reviews = reviewRepository.findByPerfumeId(perfumeId)
+        if (reviews.isEmpty()) {
+            throw CustomException(ErrorCode.REVIEW_NOT_FOUND)
+        }
```

Comment:
```suggestion
  val review = reviewRepository.findById(id)
            .orElseThrow { CustomException(ErrorCode.REVIEW_NOT_FOUND) }
```
이런식으로 수정하면 좋을것 같아요
@yeon105